(EZ-127) Add docker-build to build an image from a clojure repo (#509)
puppetlabs/ezbake: README.md

by setting the `COW` environment variable. These variables should be
space-separated lists of rpm(MOCK) and deb(COW) platforms.

#### `docker-build`

```shell
lein with-profile ezbake ezbake docker-build
```

This will build a container image using the settings in the `:docker` section

> of the `:lein-ezbake` config in `project.clj`. See the section on building

Reviewer: Would be nice as a link.
```suggestion
of the `:lein-ezbake` config in `project.clj`. See [Building container images](#building-container-images).
```

> of the `:lein-ezbake` config in `project.clj`. See [Building container images](#building-container-images).

Reviewer 2: ```suggestion
of the `:lein-ezbake` config in `project.clj`. See [Building container images](#building-container-images)
```

Reviewer: @nicklewis can you accept this change?

> container images below for more detail.

Reviewer 2: ```suggestion
below for more detail.
```

Reviewer: And this one.
